File: MJAlertViewDemo/MJAlertViewDemo/ViewController.m
//
//  ViewController.m
//  MJAlertViewDemo
//
//  Created by Mayur on 2/19/15.
//  Copyright (c) 2015 MjApps. All rights reserved.
//

#import "ViewController.h"
#import "UIView+MJAlertView.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

#pragma mark - Button actions
- (IBAction)showAlert:(id)sender {
    [UIView addMJNotifierWithText:@"This won't go. Click Dismiss to hide it!" dismissAutomatically:NO];
}
- (IBAction)showDismissableAlert:(id)sender {
    [UIView addMJNotifierWithText:@"Hey there!" dismissAutomatically:YES];
}
- (IBAction)dismissAlert:(id)sender {
    [UIView dismissMJNotifier];
}
- (IBAction)showWiderAlert:(id)sender {
    [UIView addMJNotifierWithText:@"Showing off a wider alert. #Yay" dismissAutomatically:YES];
}

@end

File: MJAlertViewDemo-Swift/MJAlertViewDemo-Swift/ViewController.swift
//
//  ViewController.swift
//  MJAlertViewDemo-Swift
//
//  Created by Jason.Chengzi on 7/15/16.
//  Copyright © 2016 Jason. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func showSuccess(sender: AnyObject) {
        UIView.makeSuccessNotificationWithText("Success Color")
    }
    
    @IBAction func showError(sender: AnyObject) {
        UIView.makeErrorNotificationWithText("Error Color")
    }
    
    @IBAction func showWarn(sender: AnyObject) {
        UIView.makeWarnNotificationWithText("Warn Color")
    }
    
    @IBAction func showInfo(sender: AnyObject) {
        UIView.makeInfoNotificationWithText("Info Color")
    }
    
}

